bl-plugins/backup/plugin.php

```diff
@@ -12,13 +12,50 @@ class pluginBackup extends Plugin {
 	// This variable define if the extension zip is loaded
 	private $zip = false;
 
+	// The last request status
+	private $last_status = null;
```

Comment:
Can you change the variables name for `lastStatus` and `lastMessage`, I'm trying to keep all variables and function name with this structure. I hope you don't mind, thank you!

Reply:
Sure, I've changed it.